tests/installation/partitioning_splitusr.pm

```diff
@@ -32,12 +32,15 @@ sub run {
     send_key "home";
     send_key_until_needlematch 'root-partition-selected', 'down';    # Select root partition
                                                                      # Different shortcut on storage-ng
-    wait_screen_change { send_key((is_storage_ng) ? 'alt-z' : $cmd{resize}) };    # Resize
-    send_key(is_storage_ng() ? 'alt-c' : 'alt-u');                                # Custom size
+    wait_screen_change { send_key((is_storage_ng) ? $cmd{resize} : 'alt-z') };    # Resize
```

Comment:
Right, but that's exactly my point, for storage-ng we have $cmd{resize} now, and for previous storage stack we had $cmd{resize}, so it should work with $cmd{resize} in both scenarios. Even we don't run it with SLE, it will be correct shortcut, as we have same package. 'alt-z' which was there before is not used anymore.

Reply:
Alright! thanks!